persistent/picklecache.py

```diff
@@ -30,12 +36,37 @@ def __init__(self, object, next=None, prev=None):
         self.next = next
         self.prev = prev
 
+def _sweeping_ring(f):
```

Comment:
Comment added, although it doesn't make this line show up as an outdated diff for some reason.